apps/src/p5lab/P5Lab.js

```diff
@@ -322,6 +324,15 @@ P5Lab.prototype.init = function(config) {
   config.enableShowLinesCount = false;
 
   const onMount = () => {
+    try {
+      const localeCode = getScriptData('appoptions').locale;
+      getStore().dispatch(setLocaleCode(localeCode));
+    } catch (exception) {
+      console.log('Got an error while trying to parse for the locale code');
+      console.log(exception);
```

Comment:
```suggestion
      console.warn('Unable to retrieve locale code, defaulting to en_us');
      console.warn(exception);
```
Use console.warn and describe what the fallback behavior is.